tests/dialect_tests.py

```diff
@@ -26,35 +26,69 @@
 sa.dialects.registry.register("pysqream.dialect", "dialect", "SqreamDialect")
 
 
-class TestSqlalchemy(TestBase):
-
-    def test_sqlalchemy(self):
+class TestDdl(TestBase):
 
+    def test_get_tables(self):
         Logger().info('SQLAlchemy direct query tests')
-        manual_conn_str = f"sqream_blue://{self.domain}:443/master"
-        connect_args = {'access_token': _access_token}
-        engine2 = create_engine(manual_conn_str, connect_args=connect_args)
-        res = engine2.execute('select 1')
-        res.fetchall()
-        assert(all(row[0] == 1 for row in res))
-
-        # Simple direct Engine query - this passes the queries to the underlying DB-API
-        res = self.engine.execute('create or replace table "kOko" ("iNts fosho" int not null)')
-        res = self.engine.execute('insert into "kOko" values (1),(2),(3),(4),(5)')
-        res = self.engine.execute('select * from "kOko"')
-        # Using the underlying DB-API fetch() functions
-        assert(res.fetchone() == ([1],))
-        assert(res.fetchmany(2) == [(2,), (3,)])
-        assert(res.fetchall() == [(4,), (5,)])
-
-        # Reflection test
-        inspector = inspect(self.engine)
-        inspected_cols = inspector.get_columns('kOko')
-        assert (inspected_cols[0]['name'] == 'iNts fosho')
-
-        self.metadata.reflect(bind=self.engine)
-        assert(repr(self.metadata.tables["kOko"]) == f"Table('kOko', MetaData(bind=Engine(sqream_blue://{self.domain}:443/master)), Column('iNts fosho', Integer(), table=<kOko>, nullable=False), schema=None)")
+        try:
```

Comment:
@zachik250 you don't need to connect because you already connected because you 
inheritor from testBase, you only need to use self.engine